zhaquirks/tuya/ts0601_motion.py

```diff
@@ -381,9 +514,43 @@ class MmwRadarMotion(CustomDevice):
                     TuyaOccupancySensing,
                     TuyaAnalogInput,
                     TuyaIlluminanceMeasurement,
+                    TuyaMmwRadarTargetDistance,
+                    TuyaMmwRadarSensitivity,
```

Comment:
The `AnalogInput` cluster ID (extended) is duplicated here.
This Tuya stuff is already somewhat of a workaround everywhere. Are both clusters working correctly here like this?

Reply:
It seems to work, yes, though in any case I now removed AnalogInput from the cluster.  Not sure if the duplication is gone: I don't think all the tests are running against this PR.